### PR TITLE
Display icon only with ADMINISTER permission

### PR DESCRIPTION
**Pull request by jonesbusy**
Accessing the page require ADMINISTER permission. 

We should also hide the button with the user doesn't have such permission

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/PurgeBuildQueueAction.java [hunk 1]
```diff
@@ -85,7 +85,12 @@ public String getDisplayName() {
 
     @Override
     public String getIconFileName() {
-        return "gear.svg";
+        if (Jenkins.get().hasPermission(Jenkins.ADMINISTER)) {
+            return "gear.svg";
+        }
+        else {
+            return null;
+        }
```
Comment (NotMyFault) [inline src/main/java/io/jenkins/plugins/PurgeBuildQueueAction.java:93]:
```suggestion
        return Jenkins.get().hasPermission(Jenkins.ADMINISTER) ? "gear.svg" : null;
```

**Review by NotMyFault — COMMENTED** [review]
But looks good overall!

**Comment by jonesbusy** [comment]
Not sure why the security scan is failing :(

**Review by NotMyFault — APPROVED** [review]
Thanks for your contribution!

**Comment by NotMyFault** [comment]
> Not sure why the security scan is failing :(

Because there was an annotation missing, I fixed that already on the default branch.

**Pull request merged**